theforeman.org/pipelines/lib/rbenv.groovy

```diff
@@ -0,0 +1,31 @@
+def bundleInstall(version, gemfile=null) {
+    command = "bundle install"
+
+    if (gemfile) {
+        command = "${command} --gemfile=${gemfile}"
+    }
+
+    withRuby(version, "bundle config set path ~/.rubygems")
+    withRuby(version, command)
+}
+
+def bundleExec(version, command, gemfile=null) {
+    command = "bundle exec ${command}"
+
+    if (gemfile) {
+        command = "BUNDLE_GEMFILE=${gemfile} bundle exec ${command}"
+    }
+
+    withRuby(version, command)
+}
+
+def withRuby(version, command) {
+    echo command.toString()
+
+    sh """
+        export PATH="/usr/lib64/rbenv/libexec/:\$HOME/.rbenv/shims:\$HOME/.rbenv/versions:\$PATH"
+        export RBENV_VERSION=${version}
+        export RBENV_ROOT="\$HOME/.rbenv"
```

Comment:
I would have expected the following is sufficient, but did not further test it:
```suggestion
        export PATH="\$HOME/.rbenv/shims:\$PATH"
        export RBENV_VERSION=${version}
```

Reply:
You are correct -- https://ci.theforeman.org/job/simple-ruby-test/28/console